js/plugins/grid.html

```diff
@@ -79,7 +79,7 @@
             var shortcodes = '[bs_row class="row"]';
             $('#demo_grid > div').each(function() {
                 $(this).removeClass('demo_col active');
-                var clss = $(this).attr('class');
+                var clss = $(this).attr('class').replace(/xs/g, "sm");
```

Comment:
Sorry, this is way late response. Once again, thanks for putting together the plugin! 

The `.replace(/xs/g, "sm")` is meant to add the "sm" class to the custom column shortcodes. I wanted to keep the column size "xs" for the visual editor when you were creating your custom columns but output a "sm" class for the shortcode. This was my hack to do that. There may be a much better way.